index.html

```diff
@@ -31,7 +31,7 @@
                 <li><a href="">About</a></li>
                 <li><a href="">Contact</a></li>
                 <li>
-                    <a href="edubew98@gmail"><img
+                    <a href="edubew98@gmail.com"><img
```

Comment:
Kindly change the contrast to be at least `4.5` of the `Send` buttons in `hover` because they failed in the contract check
![image](https://user-images.githubusercontent.com/24703171/156227753-cb124382-7083-454d-ba7d-9cef20853577.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)